streams/src/main/java/org/apache/kafka/streams/processor/internals/metrics/StreamsMetricsImpl.java

```diff
@@ -339,6 +339,17 @@ public void removeMetric(final MetricName metricName) {
         metrics.removeMetric(metricName);
     }
 
+    public void removeStoreLevelMetric(final MetricName metricName) {
```

Comment:
The original PR did add `removeMetric()` from above, which is not also used by other code, so we need to keep it, and add this new method instead.